sync/base.py

```diff
@@ -116,7 +170,7 @@ def obj_id(self):
 
     @property
     def seq_id(self):
-        return int(self._seq_id) if self._seq_id is not None else None
+        return int(self._seq_id) if self._seq_id is not None else 0
```

Comment:
Why not just make `_seq_id` 0 in the `__init__` and forget the None. It seems kind of weird to me that in this case the `_cache_key` and `key` are different.